#There ara some params to be loaded.

frame_id: neuvition
data_mode: 0
pwm_value: 55
#0:200KHZ  1:340HZ 2:500KHZ  3:750HZ 4:1MHZ
laser_period: 2
#0:5fps  1:6fps 2:10fps  3:15fps 4:30fps
data_frame: 0
#0:PolyChrome  1:Camera
color_mode: 0
upstream_port: 6668
upstream_host: 192.168.1.101
#camera  true:open false:close
video_fusion: true
# filter 1:gavin filters 0:frank filters
laser_filters: 1
# image 0:no swap  1: swap
laser_image: 0
laser_time: 0



